axelrod/tests/strategies/test_backstabber.py

```diff
@@ -24,58 +24,100 @@ def test_strategy(self):
         Forgives the first 3 defections but on the fourth
         will defect forever. Defects after the 198th round unconditionally.
         """
+        self._defects_after_four_defections()
```

Comment:
Remove these and change the name of the methods to be `test_defects_after_four_defections`.